Lab4/manifest.result.yaml

```diff
@@ -17,25 +17,25 @@ spec:
     spec:
       containers:
       - name: petstore
-        image: {REPLACE_WITH_ECR_REGISTRY}/petstore
+        image: christopherhein/petstore:latest
         ports:
         - name: http-server
           containerPort: 8080
         - name: wildfly-cord
           containerPort: 9990
         env:
         - name: DB_URL
-          value: "jdbc:postgresql://{REPLACE_WITH_RDS_URL}:5432/petstore?ApplicationName=applicationPetstore"
+          value: "jdbc:postgresql://container-migration-workshop.cwlpneewkyvy.us-west-1.rds.amazonaws.com:5432/petstore?ApplicationName=applicationPetstore"
```

Comment:
I think you accidentally overwrote the placeholder...?

Reply:
I think I did too, hopefully trolls found this and went ham on my db lol